pkg/api/types.go

```diff
@@ -47,12 +47,16 @@ func ProtoLvolToLvol(l *spdkrpc.Lvol) *Lvol {
 	if l == nil {
 		return nil
 	}
+	parent := l.Parent
+	if types.IsBackingImageSnapLvolName(parent) {
```

Comment:
Should this be `if types.IsBackingImageSnapLvolName(l.Name)`?

Reply:
`l.Name` is the name of the replica that is a child of the backing image, and `l.Parent` refers to the `base_snapshot`, which is the name of the backing image.

For v1 volumes, the replica that immediately follows the backing image should have an empty `Parent` field. Therefore, if the `base_snapshot` is the backing image, I set `l.Parent` to an empty string.

Reply:
Got it. I was thinking that this PR will unset `parent` field of a backing image lvol.